stable/velero/values.yaml

```diff
@@ -19,16 +19,14 @@ podAnnotations: {}
 # Resource requests/limits to specify for the Velero deployment. Optional.
 resources: {}
 
-# Pod priority class name to use for the Velero deployment. Optional.
-priorityClassName: {}
-
-# Init containers to add to the Velero deployment's pod spec. Optional.
+# Init containers to add to the Velero deployment's pod spec. At least one plugin provider image is required.
 initContainers: []
-  # - name:
-  #   image:
+  # - name: velero-plugin-for-aws
+  #   image: velero/velero-plugin-for-aws:v1.0.0-beta.1
```

Comment:
```suggestion
  #   image: velero/velero-plugin-for-aws:v1.0.0
```

Reply:
where do I need to specify the plugins inside of the helm chart. Is there a reference to the values file? Actualy im having an issue upgrading.
https://github.com/helm/charts/issues/19427

Reply:
You have to specify it in the `initContainers` field in the values file like is exemplified [here](https://github.com/carlisia/charts/blob/c-velero-1.2.0-beta/stable/velero/values.yaml#L23). Hope this was the answer you were looking for.

Reply:
I could solve the problem by using init containers:
```
initContainers:
  - name: velero-plugin-for-microsoft-azure
    image: velero/velero-plugin-for-microsoft-azure:v1.0.0
    volumeMounts:
      - name: plugins
        mountPath: /target
```

Reply:
@JoaoBraveCoding yes, thats it. I think I didn´t see your message on Friday, but today. Thank you!